token.go

```diff
@@ -45,7 +46,8 @@ func NewWithClaims(method SigningMethod, claims Claims) *Token {
 	}
 }
 
-// SignedString retrieves the complete, signed token
+// SignedString creates and returns a complete, signed JWT token.
```

Comment:
```suggestion
// SignedString creates and returns a complete, signed JWT.
```

JWT token is redundant ;)